### DISCUSSION TITLE
v1.8.1

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#293] [Release]
**Features:**
Localize masterserver errors ([#333](https://github.com/R2Northstar/NorthstarMods/pull/333), [#159](https://github.com/R2Northstar/NorthstarLauncher/pull/159))
Include server description in search ([#388](https://github.com/R2Northstar/NorthstarMods/pull/388))
Expose IsDedicated on server vm ([#179](https://github.com/R2Northstar/NorthstarLauncher/pull/179))

**Fixes:**
Guard against GetEntByIndex read out of bounds ([#191](https://github.com/R2Northstar/NorthstarLauncher/pull/191))
Fix segfault on badly formed compressed packets ([#160](https://github.com/R2Northstar/NorthstarLauncher/pull/160))
Fix bug where emit wouldn't be blocked if uppercase ([#192](https://github.com/R2Northstar/NorthstarLauncher/pull/192))
Unlock SP when Coop is selected ([#383](https://github.com/R2Northstar/NorthstarMods/pull/383)) 
Attempt reconnect in case of DUPLICATE_SERVER ([#187](https://github.com/R2Northstar/NorthstarLauncher/pull/187))
Set default val for net_chan_limit_msec_per_sec ([#196](https://github.com/R2Northstar/NorthstarLauncher/pull/196))
Open `placeholder_playerdata` as an input stream ([#173](https://github.com/R2Northstar/NorthstarLauncher/pull/173))
LatencyFleX Implementation Update ([#183](https://github.com/R2Northstar/NorthstarLauncher/pull/183))
Fix "Auth failed" when trying to play singleplayer ([#184](https://github.com/R2Northstar/NorthstarLauncher/pull/184))

**Contributors:**
@cpdt @emma-miler @F1F7Y @GeckoEidechse @KittenPopo @KyleGospo @pg9182 @RoyalBlue1

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.8.1'>v1.8.1</a>.</em>

**Comment by Kreanch** [comment]
Можно вдвоём проходить сюжет или я что-то не так понял.

**Reply by GeckoEidechse** [reply]
No, coop isn't publicly released yet and still very much WIP as it crashes in a lot of places. Will sadly take a bit until it's ready for release :/

**Reply by Kreanch** [reply]
Хорошо я буду ждать, спасибо. XP
